src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,441 @@
+            }
+        }
+
+        /// <inheritdoc/>
+        protected virtual void Dispose(DisposeTypes disposing)
+        {
+            int res;
+            if (!disposed)
+            {
+                if (_tzshQpClient != IntPtr.Zero)
+                {
+                    res = Interop.QuickPanelClient.Destroy(_tzshQpClient);
+                    try
+                    {
+                        _tzsh.ErrorCodeThrow(res);
+                    }
+                    catch (ArgumentException e)
+                    {
+                        throw new MemberAccessException("QuickPanelClient is a corrupted");
+                    }
+                    _tzshQpClient = IntPtr.Zero;
+                }
+                disposed = true;
+            }
+        }
+
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        private void OnVisibleChanged(int type, IntPtr ev_info, IntPtr data)
+        {
+            int res;
+            int state;
+
+            res = Interop.QuickPanelClient.GetEventVisible(ev_info, out state);
+            _tzsh.ErrorCodeThrow(res);
+
+            _visibleChanged?.Invoke(this, (StateVisible)state);
+        }
+
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        private void OnOrientationChanged(int type, IntPtr ev_info, IntPtr data)
+        {
+            int res;
+            int state;
+
+            res = Interop.QuickPanelClient.GetEventOrientation(ev_info, out state);
+            _tzsh.ErrorCodeThrow(res);
+
+            _orientationChanged?.Invoke(this, (StateOrientation)state);
+        }
+
+        /// <summary>
+        /// Gets the visible state of the quickpanel.
+        /// </summary>
+        /// <returns>The visible state of the quickpanel service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public StateVisible GetVisible()
+        {
+            int res;
+            int vis;
+
+            res = Interop.QuickPanelClient.GetVisible(_tzshQpClient, out vis);
+
+            _tzsh.ErrorCodeThrow(res);
+
+            return (StateVisible)vis;
+        }
+
+        /// <summary>
+        /// Gets the window's scrollable state of the quickpanel service window.
+        /// </summary>
+        /// <returns>The scrollable state of the quickpanel service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public StateScrollable GetScrollable()
+        {
```

Comment:
same as above...
How about making this as get only property?

Reply:
I changed this to property named Scrollable